website/content/installation/_index.md

```diff
@@ -94,7 +94,6 @@ service account. It is suggested to use either the charts or kustomize if they
 need to be changed.
 
 {{% notice note %}}
-
```

Comment:
is this related to the issue?

Reply:
@fedepaol  - no this is not related to the issue but I'm not 100% sure. This is more of changing it to match the other `notice note` tags. If you look at the other `{{% notice note %}}` tags, you'll see that the text following it immediately begins in the new line while in this case there was an extra new line. Feel free to correct me if I'm wrong here.